app/lib/hesa/parsers/itt_record.rb

```diff
@@ -28,16 +28,13 @@ class IttRecord
         end_date: "F_ENDDATE",
         reason_for_leaving: "F_RSNEND",
         bursary_level: "F_BURSLEV",
-        trn: "F_TREFNO",
+        trn: "F_TRN",
         training_route: "F_ENTRYRTE",
         nationality: "F_NATION",
         hesa_updated_at: "F_STATUS_TIMESTAMP",
         itt_aim: "F_ITTAIM",
         itt_qualification_aim: "F_QLAIM",
         fund_code: "F_FUNDCODE",
-        study_length: "F_SPLENGTH",
```

Comment:
Should we be getting rid of `F_SPLENGTH`, `F_UNITLGTH` and `F_SERLEAVE`  because it's for the `hesa_metadata` table which was created for the finance team. They might be still interested in this information. Worth speaking to Andrew to double check we're ok to remove as Andrew and I worked together to create this table.